src/Entity/User.php

```diff
@@ -50,6 +51,12 @@ class User
      */
     private Collection $chats;
 
+    #[ORM\Column(type: 'boolean', nullable: true)]
+    private bool $isVerified;
```

Comment:
I think you should set this property like :
```suggestion
    private bool $isVerified = false;
```